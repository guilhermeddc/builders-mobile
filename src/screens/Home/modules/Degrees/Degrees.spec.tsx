import React from 'react';
import {Degrees} from '.';
import {render} from '@testing-library/react-native';

describe('Degrees', () => {
  it('should render correctly', () => {
    const tree = render(<Degrees temp={0} />);
    expect(tree.toJSON()).toMatchSnapshot();
    expect(tree.getByText(/-273.15/i)).toBeDefined();
  });
});
