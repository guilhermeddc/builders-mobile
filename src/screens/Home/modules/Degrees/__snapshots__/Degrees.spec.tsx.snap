// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Degrees should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#0d0c9a",
        "borderRadius": 125,
        "height": 250,
        "justifyContent": "center",
        "marginTop": 32,
        "width": 250,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontSize": 54,
          "fontWeight": "400",
          "textAlign": "center",
        },
      ]
    }
  >
    -273.15
    ËšC
  </Text>
</View>
`;
